validator/keymanager/remote/mock/mock_keymanager.go

```diff
@@ -33,8 +37,19 @@ func (m *MockKeymanager) FetchValidatingPublicKeys(context.Context) ([][fieldpar
 }
 
 // Sign --
-func (*MockKeymanager) Sign(context.Context, *validatorpb.SignRequest) (bls.Signature, error) {
-	panic("implement me")
+func (*MockKeymanager) Sign(_ context.Context, s *validatorpb.SignRequest) (bls.Signature, error) {
+	key, err := bls.RandKey()
+	if err != nil {
+		return nil, err
+	}
+	st, _ := util.DeterministicGenesisState(nil, 1)
+	d := params.BeaconConfig().DomainApplicationBuilder
```

Comment:
It's a little bad to hard code domain here IMO. The next person that uses it might not be happy. Couldn't you just use `s.SignatureDomain` from the parameter?

Reply:
hadn't realized this will update